src/handlers/summary.ts

```diff
@@ -9,7 +9,7 @@ export interface ResultInfo {
 
 function generateGitHubSummary(context: Context, urls: ResultInfo[]): string {
   const target = `https://github.com/${context.payload.repository.owner?.login}`;
-  const output: string[] = ["## Merge report\n\n"];
+  const output: (string | undefined)[] = ["## Merge report\n\n"];
```

Comment:
We should filter out empty elements before placing into the array probably.

Reply:
This is just to satisfy TypeScript. Due to the nature of JSON7 I had to make these values optional but in fact they will never be undefined because they contain a default. But if optional is not set then the configuration will complain that the types are required in the configuration which is not the case. I couldn't find a better way to solve this.